cmd/src/campaigns_list.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"time"
+)
+
+func init() {
+	usage := `
+Examples:
+
+  List campaigns (default limit is 1000):
+
+    	$ src campaigns list
+
+  List only the first 5 campaigns:
+
+    	$ src campaigns list -first=5
+
+  List campaigns and only print their IDs (default is to print ID and Name):
+
+    	$ src campaigns list -first=5 -f '{{.ID}}'
+
+`
+
+	flagSet := flag.NewFlagSet("list", flag.ExitOnError)
+	usageFunc := func() {
+		fmt.Fprintf(flag.CommandLine.Output(), "Usage of 'src repos %s':\n", flagSet.Name())
+		flagSet.PrintDefaults()
+		fmt.Println(usage)
+	}
+	var (
+		firstFlag  = flagSet.Int("first", 1000, "Returns the first n repositories from the list. (use -1 for unlimited)")
+		formatFlag = flagSet.String("f", "{{.ID}}: {{.Name}}", `Format for the output, using the syntax of Go package text/template. (e.g. "{{.ID}}: {{.Name}}") or "{{.|json}}")`)
+		apiFlags   = newAPIFlags(flagSet)
+	)
+
+	handler := func(args []string) error {
+		flagSet.Parse(args)
+
+		tmpl, err := parseTemplate(*formatFlag)
+		if err != nil {
+			return err
+		}
+
+		query := `query Campaigns($first: Int) {
+  campaigns(first: $first) {
+    nodes {
+      id
+      name
+      description
+      createdAt
+      updatedAt
+
+      changesets {
```

Comment:
Should we pass a `first` pagination parameter here too?

Reply:
Good idea. I'll do that.